sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -231,16 +234,17 @@ class Dataset[T] private[sql](
   }
 
   /**
-   * Compose the string representing rows for output
+   * Get rows represented in Sequence by specific truncate and vertical requirement.
    *
-   * @param _numRows Number of rows to show
+   * @param numRows Number of rows to return
    * @param truncate If set to more than 0, truncates strings to `truncate` characters and
    *                   all cells will be aligned right.
-   * @param vertical If set to true, prints output rows vertically (one line per column value).
+   * @param vertical If set to true, the rows to return don't need truncate.
```

Comment:
I would avoid abbreviation in the documentation. `don't` -> `do not`.

Reply:
Yep, all abbreviation done in d4bf01a.